packages/common/src/services/integrations/jwp/JWPCheckoutService.ts

```diff
@@ -23,19 +25,21 @@ import type { ServiceResponse } from '../../../../types/service';
 
 import type {
   CommonResponse,
-  GetAccessFeesResponse,
-  AccessFee,
   MerchantPaymentMethod,
   GeneratePayPalParameters,
   VoucherDiscountPrice,
   GetItemAccessResponse,
+  StripeProduct,
+  StripePrice,
 } from './types';
 import JWPAPIService from './JWPAPIService';
 
 @injectable()
 export default class JWPCheckoutService extends CheckoutService {
   private readonly cardPaymentProvider = 'stripe';
 
+  accessMethod: AccessMethod = 'plan';
```

Comment:
Do we need this?

Reply:
Well, yes
![image](https://github.com/user-attachments/assets/5b224a16-4ce6-4cb5-a997-15bfd482945a)

Reply:
We don't need additional setter to determine that, as we will have to introduce one for the access bridge. 

If the environment variable for the Access Bridge is defined, we know for sure that the new flow will be used. It can be part of the config store even before initalization. 

I had similar case here, so I was thinking something like this that we can reuse throughout entire app, and not just on the payment side.
https://github.com/jwplayer/ott-web-app/pull/603/files#diff-5249be0266da3296ee9e5a0f29405b5d56bc6f629025e88fb421964ec0edff70R49